docs/migration-guides/1.20-1.21.md

```diff
@@ -0,0 +1,81 @@
+# Migrate from 1.20 to 1.21
+
+## Eventing
+In Kyma 1.21.0, the eventing component has been switched completely from Knative to Kyma internal Subscription custom resource which is based on Nats server. In case the subscribers fail to get triggered by events, check the status of the subscriptions:
```

Comment:
```suggestion
In Kyma 1.21.0, the Eventing component has been switched completely from Knative to Kyma internal Subscription Custom Resource which is based on NATS server. 

In case the subscribers fail to get triggered by events, check the status of the subscriptions:
```